doc/configure

```diff
@@ -206,6 +206,10 @@ def main():
 
     add_litani_7(builds, html_mans, args.gen_html)
 
+    for h in html_mans:
+        if "litani.7" in str(h.resolve()):
+            html_mans.insert(0, html_mans.pop(html_mans.index(h)))
+
```

Comment:
This change is made in order to bring `litani.7` at top of `html_mans` list so that it can be parsed firat and stay first in report navigation.

Reply:
So what you're saying is that `litani.7` is not the name of a page? Is it a directory or something?

Reply:
Would it then make sense to say at least `if "litani.7" in h.name`? That way you don't need to search through the entire path from the root to the file

Reply:
Or why don't you say even `if "litani.7.html" == h.name?`

Reply:
Makes sense. Fixed.